packages/button/src/Button.story.tsx

```diff
@@ -45,6 +46,11 @@ storiesOf('Buttons', module)
       { div: 'div', span: 'span', button: 'button' },
       'div',
     );
+    const selectedGlyph = select(
```

Comment:
I know that @DesignerDave might disagree with me on this one, but think it could be worth either breaking this out into its own story or bringing this logic into the `default` tab. This behavior isn't related to whether or not it's a custom component, so I think it might be confusing to group this here.

Reply:
Still think it makes sense to move this out of custom component. Can be in glyph or it's own story up to you

Reply:
ok, this logic has now been moved to the default tab

Reply:
👍